Skype/WebSDK/docs/PTVideoGroup.md

```diff
@@ -6,83 +6,113 @@
 
-1. Start group video conversation, and track participant events 
+    ```js
+        conversation.selfParticipant.video.state.when('Connected', function () {
+            // video is availabe ... lets assign a container.
+            conversation.selfParticipant.video.channels(0).stream.source.sink.format('Stretch'); // formats include: 'Stretch', 'Fit' and 'Crop'
+            conversation.selfParticipant.video.channels(0).stream.source.sink.container(/* DOM node such as a DIV */);
+            // the video will be rendered automatically
+        });
+    ```
+1. To render the video of the other participants in the conversation we add another listener that notifies us when
+another participant joins the conversation.
+
+    > [!Warning]
+    > Group video for **Google Chrome** is rendered using a different API.
+    > Please see [Group in Google Chrome](PTVideoGroupGoogleChrome.md) for more details.
 
-  ```js
-    var conversationsManager = application.conversationsManager;
-    var id = content.querySelector('.id').value;
-    var id2 = content.querySelector('.id2').value;
-    conversation = conversationsManager.createConversation();
+    ```js
+        conversation.participants.added(function (person) {
+            // person.displayName() has joined the conversation
 
-    function setupContainer(person, size, videoDiv) {
-        person.video.channels(0).stream.source.sink.format('Stretch');
-        person.video.channels(0).stream.source.sink.container(videoDiv);
-    }
+            // lets add another listener for the joined person that notifies us when they add video
+            person.video.state.when('Connected', function () {
+                // video is availabe ... lets assign a container.
 
-    listeners.push(conversation.selfParticipant.video.state.when('Connected', function () {
-        // set up local video container
-        setupContainer(conversation.selfParticipant, 'large', content.querySelector('.selfVideoContainer'));
+                // formats include: 'Stretch', 'Fit' and 'Crop'
+                person.video.channels(0).stream.source.sink.format('Stretch');
+                person.video.channels(0).stream.source.sink.container(/* DOM node such as a DIV */);
 
-        listeners.push(conversation.participants.added(function (person) {
-            // person.displayName() has joined the conversation
-            listeners.push(person.video.state.when('Connected', function () {
-                // set up remote video container
-                if (Object.keys(videoMap).length === 1) {
-                    videoMap[person.displayName()] = 2;
-                    setupContainer(person, 'large', content.querySelector('.remoteVideoContainer2'));
-                }
-                else {
-                    videoMap[person.displayName()] = 1;
-                    setupContainer(person, 'large', content.querySelector('.remoteVideoContainer1'));
-                }
+                // now we can render it
```

Comment:
Not your change, but I think this is wrong. We need to add another listener for isVideoOn, and set isStarted(true/false) when isVideoOn() becomes true/false respectively